src/ui/theme/index.ts

```diff
@@ -1,26 +1,29 @@
 import { extendTheme } from '@chakra-ui/react';
-import Button from './components/button';
 import Text from './components/text';
 import colors from './foundations/colors';
 import nyplTheme from '../nypl-base-theme';
+import { ColorMode } from '../../types';
+import { getColor } from '../../utils/getColor';
+import getButtonStyle from './components/button';
 
 /**
  * See Chakra default theme for shape of theme object:
  * https://github.com/chakra-ui/chakra-ui/tree/main/packages/theme
  */
```

Comment:
Maybe add a comment here explaining this is a function because we need to pass in the colorMode and adapt the theme to fit